web-registry/src/pages/Dashboard/MyBridge/MyBridge.BridgedTable.tsx

```diff
@@ -0,0 +1,188 @@
+import { useState } from 'react';
+import { Box } from '@mui/material';
+import { quantityFormatNumberOptions } from 'config/decimals';
+import { random } from 'lodash';
+import { ELLIPSIS_COLUMN_WIDTH, tableStyles } from 'styles/table';
+
+import { BlockContent } from 'web-components/lib/components/block-content';
+import EmptyCartIcon from 'web-components/lib/components/icons/EmptyCartIcon';
+import ErrorIcon from 'web-components/lib/components/icons/ErrorIcon';
+import PendingIcon from 'web-components/lib/components/icons/PendingIcon';
+import SuccessIcon from 'web-components/lib/components/icons/SuccessIcon';
+import InfoLabel from 'web-components/lib/components/info-label';
+import {
+  ActionsTable,
+  DEFAULT_ROWS_PER_PAGE,
+  TablePaginationParams,
+} from 'web-components/lib/components/table/ActionsTable';
+import InfoTooltipWithIcon from 'web-components/lib/components/tooltip/InfoTooltipWithIcon';
+import { formatDate, formatNumber } from 'web-components/lib/utils/format';
+
+import { useLedger } from 'ledger';
+
+import { AccountLink, Link } from 'components/atoms';
+import WithLoader from 'components/atoms/WithLoader';
+import { NoCredits } from 'components/molecules';
+import { useEcocredits } from 'hooks';
+
+import {
+  AMOUNT_BRIDGED_TOOLTIP,
+  BRIDGED_STATUS,
+  CREDIT_BATCH_TOOLTIP,
+  NO_BRIDGED_CREDITS,
+} from './MyBridge.constants';
+import { WithLoaderStyles } from './MyBridge.styles';
+import { BreakText, BreakTextEnd, GreyText } from './MyBridge.utils';
+
+const iconStyle = { height: '16px', verticalAlign: 'middle', mb: '2px' };
+
+export const BridgedTable = (): JSX.Element => {
+  const { wallet } = useLedger();
+
+  const [paginationParams, setPaginationParams] =
+    useState<TablePaginationParams>({
+      page: 0,
+      rowsPerPage: DEFAULT_ROWS_PER_PAGE,
+      offset: 0,
+    });
+
+  const { credits, isLoadingCredits } = useEcocredits({
+    address: wallet?.address,
+    paginationParams,
+  });
+
```

Comment:
I'm still not sure how we will query for these. I'm asking on Discord.

Reply:
Thanks!